README.md

```diff
@@ -9,6 +9,94 @@ Refer to the [user manual](https://veecle.github.io/veecle-os/user-manual/) to l
 Refer to [CONTRIBUTING](CONTRIBUTING.md) for build instructions and other development material.
 After completing the setup instructions, go to [the examples](veecle-os-examples/) to run some Veecle OS example programs.
 
+## Example
+
+Add the following dependencies to your `Cargo.toml` file.
+
+<!-- Dependencies in `readme-example/Cargo.toml` must be kept in sync with this. -->
+
+```toml
+rand = "0.9.2"
+tokio = "1.48.0"
+# TODO(#161): Uses a git dependency until the `ConsolePrettyExporter` is released.
+veecle-os = { git = "https://github.com/veecle/veecle-os", branch = "main", features = ["telemetry-enable", "osal-std"] }
+```
+
+Add the following code to your `main.rs` file.
+
+```rust
+use core::convert::Infallible;
+use veecle_os::info;
+use veecle_os::osal::api::time::{Duration, Instant, Interval, TimeAbstraction};
+use veecle_os::runtime::{InitializedReader, Storable, Writer};
+use veecle_os::telemetry::collector::{ConsolePrettyExporter, set_exporter, ProcessId};
+
+#[derive(Debug, Storable)]
+#[storable(data_type = "Instant")]
+pub struct Tick;
+
+/// Emits a timestamp every second.
+///
+/// Can be used on any supported platform.
+#[veecle_os::runtime::actor]
+pub async fn ticker_actor<T>(
+    mut tick_writer: Writer<'_, Tick>,
+) -> Result<Infallible, veecle_os::osal::api::Error>
+where
+    T: TimeAbstraction,
+{
+    let mut interval = T::interval(Duration::from_secs(1));
+
+    loop {
+        interval.tick().await?;
+        tick_writer.write(T::now()).await;
+    }
+}
+
+/// Prints every received tick via `veecle-telemetry` .
+///
+/// Can be used on any supported platform.
+#[veecle_os::runtime::actor]
+pub async fn ticker_reader(mut reader: InitializedReader<'_, Tick>) -> Infallible {
+    loop {
+        reader.wait_for_update().await.read(|tick| {
+            info!(
+                "latest tick",
+                tick = {
+                    i64::try_from(tick.duration_since(Instant::MIN).unwrap().as_secs()).unwrap()
+                }
+            );
+            // The example doesn't terminate on its own.
+            // To test the example in CI, we use the `exit-example` feature to terminate.
+            // When testing the example, the feature will lead to the following expected error, that can be ignored:
+            //      "unexpected `cfg` condition value: `exit-example`"
+            // To get avoid the error, please remove the following block for you code.
+            {
+                #[cfg(feature = "exit-example")]
```

Comment:
This can probably be `#[cfg(test)]`, I hope that is set within a doctest when it is compiled (if not, maybe `#[cfg(doctest)]`).

Reply:
Sadly, neither `#[cfg(test)]` nor `#[cfg(doctest)]` work for this. That's why I introduced the feature in the first place.